scripts/dap-ami-bootstrap.sh

```diff
@@ -1,38 +1,42 @@
 #!/bin/bash
 set -e
 
-if ! docker images | grep conjur; then
-  echo "Loading Conjur appliance image into Docker"
-  gzip -df /home/core/conjur-appliance.tar.gz # Unzip the tar.gz into a tar
-  docker load -i /home/core/conjur-appliance.tar
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
+
+if ! docker images | grep appliance; then
+  echo "Loading DAP appliance image into Docker"
+  docker load -i /opt/dap-appliance.tar.gz
 fi
 
 image_id=$(docker images -q)
-container_name='conjur-appliance'
+container_name='dap-appliance'
 
 docker rm -f ${container_name} > /dev/null 2>&1 || true
 
 mkdir -p /var/log/conjur
 mkdir -p /opt/conjur/backup
```

Comment:
I've been working in the [DAP Wiki](https://github.com/cyberark/dap-wiki/blob/master/tutorials/basic-cluster.md#step-2---setup-the-master-node) to provide a standard set of folders and configurations.  Instead of this backup folder, can we create the following:

```
mkdir -p /opt/cyberark/dap/configuration
mkdir -p /opt/cyberark/dap/security
mkdir -p /opt/cyberark/dap/backup
```

Reply:
Can we add a seccomp file ([example here](https://github.com/conjurdemos/dap-intro/blob/master/seccomp.json)) to the `/opt/cyberark/dap/security` folder?

Reply:
In the `/opt/cyberark/dap/configuration` folder, can we include an empty config file called `dap.json`.  It should container:
```
# dap.json
{}
```

Reply:
Seccomp is added. I've added all these dirs and their mounts. I'm going to hold off on the dap.json since you can't comment a json file and near as I can tell this is undocumented as yet.